### PR TITLE
#7515: moved to_layout to C++

### PR DESCRIPTION
**Pull request by arakhmati**
#7515

### CODE REVIEW

File: tests/scripts/run_performance.sh [hunk 1]
```diff
@@ -156,6 +156,7 @@ main() {
         exit 1
     fi
 
+    export TTNN_CONFIG_OVERRIDES='{"enable_fast_runtime_mode": true}';
```
Comment (tt-rkim) [inline tests/scripts/run_performance.sh:159]: Is this going to be a permanent option?

**Comment by tt-rkim** [comment]
sorry for issues in perf pipelines
we are fixing

File: tests/scripts/run_performance.sh (hunk 1 above)
Comment (arakhmati) [reply tests/scripts/run_performance.sh:165]: Yes, for a while. We do want to set it as a default option in the future

Comment (tt-rkim) [reply tests/scripts/run_performance.sh:165]: Did you want to try putting this as a pipeline level env var in the workflow?

**Review by eyonland — APPROVED** [review]

File: tests/scripts/run_performance.sh (hunk 1 above)
Comment (tt-rkim) [reply tests/scripts/run_performance.sh:165]: @tapspatel Putting this as a workflow level env var in all perf workflows

**Review by tt-rkim — APPROVED** [review]

**Review by tt-nshanker — APPROVED** [review]

**Pull request merged**